config/trainer_config.yaml

```diff
@@ -174,11 +190,14 @@ WalkerLearning:
     time_horizon: 1000
     batch_size: 2048
     buffer_size: 20480
-    gamma: 0.995
```

Comment:
Yeah, the old versions of the config aren't compatible. Having `gamma` won't break anything, but it will end up using the default `gamma` from `default_config`. We could auto-assign the `gamma` here to the extrinsic gamma, but that would break the abstraction. I guess we'll just have to be careful in the migration guide.